### DISCUSSION TITLE
Short-read data

### DISCUSSION THREAD

**Discussion by ezecalvo** [GoekeLab/sg-nex-data#12] [Q&A] [answered]
Hi,

I'm trying to understand which files belong to short-read seq data. Is it the ones that say "cDNA" or "cDNAStranded"? When looking at read lengths both types of files have reads longer than 300nt. So not sure about how to distinguish PCR-cDNA and short-read cDNA.

Thanks

**Comment by jonathangoeke** [comment]
Hi @ezecalvo all of these files are long read data. We plan to update the repository soon to include the links for the short read data.

**Comment by savytskanatalia** [comment]
Hello @jonathangoeke ! Any updates on the progress with update? We are also interested in short data for a small benchmark :)

**Comment by SamBryce-Smith** [comment]
Would also like to chime in and say I'd be very interested in the short read data being publicly available. Thanks for providing the long-read data already though!

**Comment by jonathangoeke** [comment, marked as answer]
Hi @ezecalvo @savytskanatalia @SamBryce-Smith 
We have just updated the SG-NEx data release to 0.2 (see #15). The short read data should now be available.
Thanks!
